crates/astria-composer/local.env.example

```diff
@@ -69,6 +69,9 @@ ASTRIA_COMPOSER_METRICS_HTTP_LISTENER_ADDR="127.0.0.1:9000"
 # The address at which the gRPC collector and health services are listening.
 ASTRIA_COMPOSER_GRPC_ADDR="0.0.0.0:0"
 
+# The asset to use for paying for transactions submitted to sequencer.
+ASTRIA_COMPOSER_FEE_ASSET="nria"
```

Comment:
Note that this sets the payment fee for all actions to `nria` - the same as was previously hard coded. This might not be exactly what we want to do in the future.